src/libraries/System.Security.Cryptography.Xml/tests/EncryptedXmlTest.cs

```diff
@@ -145,7 +145,8 @@ public void Sample2()
             {
                 aes.Mode = CipherMode.CBC;
                 aes.KeySize = 256;
-                aes.Key = Convert.FromBase64String("o/ilseZu+keLBBWGGPlUHweqxIPc4gzZEFWr2nBt640=");
+                byte[] disposableValue = Convert.FromBase64String("o/ilseZu+keLBBWGGPlUHweqxIPc4gzZEFWr2nBt640=");
```

Comment:
I was about `disposableKey` or `disposableSecret` but both was recognised as a suspicious keyword.

Reply:
I worry that this is a non-sustainable/non-discoverable pattern.  Would it instead work to do

```C#
aes.Key = Convert.FromBase64String(TestSecretHelper.Whatever("o/ilseZu+keLBBWGGPlUHweqxIPc4gzZEFWr2nBt640="));
```

Reply:
unfortunately no. The `.Key` is trigger there and the tool recognise base64 for as a key in any context.

Another option would be `// [SuppressMessage ...]`.